src/Type/NullType.php

```diff
@@ -115,6 +115,16 @@ public function getIterableValueType(): Type
 		return new ErrorType();
 	}
 
+	public function isOffsetAccesible(): TrinaryLogic
+	{
+		return TrinaryLogic::createNo();
+	}
+
+	public function getOffsetValueType(): Type
+	{
+		return new ErrorType();
```

Comment:
Even though we return No in `isOffsetAccesible` we should probably return null type here as it is always known.

Reply:
fixed